ieeeuoft/hackathon-template
# updated hardware model to accept blank/null fields

QA Steps:
- run migration
- go to admin site and you should be able to change most fields in your hardware to be empty and save successfully
